charts/karpenter/templates/deployment.yaml

```diff
@@ -73,8 +73,15 @@ spec:
               value: "1.19.0-0"
             - name: KARPENTER_SERVICE
               value: {{ include "karpenter.fullname" . }}
+          {{- if .Values.webhook.enabled }}
             - name: WEBHOOK_PORT
               value: "{{ .Values.webhook.port }}"
+            - name: WEBHOOK_METRICS_PORT
+              value: "{{ .Values.webhook.metrics.port }}"
+          {{- else }}
+            - name: DISABLE_WEBHOOK
+              value: "true"
```

Comment:
Do we even need this? Where does Karpenter reference this? Shouldn't that be included in this PR ?

Reply:
Discussed offline. This is taken through karpenter-core to selectively enable the webhooks in the binary.